projects/packages/my-jetpack/_inc/data/utils/get-guessed-site-lifecycle-status.ts

```diff
@@ -0,0 +1,35 @@
+const getGuessedSiteLifecycleStatus = () => {
+	const lifecycleStats = window?.myJetpackInitialState?.lifecycleStats;
+
+	const {
+		modules,
+		purchases,
+		jetpackPlugins: plugins,
```

Comment:
I noticed the actual name for this key in state is `jetpackPlugins`, something we missed when reviewing the addition of this function

Reply:
Thanks for fixing that!